package.json

```diff
@@ -2,9 +2,9 @@
   "name": "jqtodo",
   "version": "1.0.0",
   "description": "",
-  "main": "index.js",
+  "main": "./src/server.js",
```

Comment:
this should point at the entry point of your app, or remove it. Don't leave invalid config around.